packages/paypal-commerce-integration/src/paypal-commerce-credit-card/paypal-commerce-credit-cards-strategy.spec.ts

```diff
@@ -282,115 +243,9 @@ describe('PayPalCommerceCreditCardsPaymentStrategy', () => {
-
             await strategy.initialize(initializationOptionsWithStyles);
 
-            expect(paypalSdk.HostedFields.render).toHaveBeenCalledWith({
-                fields: expectedCreditCardFormFields,
-                styles: {
-                    input: {
-                        color: 'gray',
-                        'font-family': 'bigFont',
-                        'font-size': '14px',
-                        'font-weight': '400',
-                    },
-                    '.invalid': {
-                        color: 'red',
-                        'font-family': 'bigFont',
-                        'font-size': '14px',
-                        'font-weight': '400',
-                    },
-                    ':focus': {
-                        color: 'black',
-                        'font-family': 'bigFont',
-                        'font-size': '14px',
-                        'font-weight': '400',
-                    },
-                },
-                paymentsSDK: true,
-                createOrder: expect.any(Function),
-            });
-        });
-
-        it('sets hosted form events if related callbacks were provided', async () => {
-            const hostedFieldOn = jest.fn();
-
-            jest.spyOn(paypalSdk.HostedFields, 'render').mockImplementation(() =>
-                Promise.resolve({
-                    on: hostedFieldOn,
-                }),
-            );
-
-            const initializationOptionsWithStyles = {
-                methodId,
-                paypalcommercecreditcards: {
-                    form: {
-                        fields: paypalCommerceCreditCardsOptions.form.fields,
-                        onBlur: jest.fn(),
-                        onFocus: jest.fn(),
-                        onEnter: jest.fn(),
-                        onCardTypeChange: jest.fn(),
-                        onValidate: jest.fn(),
-                    },
-                },
-            };
-
-            await strategy.initialize(initializationOptionsWithStyles);
-
-            expect(hostedFieldOn).toHaveBeenCalledWith('blur', expect.any(Function));
-            expect(hostedFieldOn).toHaveBeenCalledWith('focus', expect.any(Function));
-            expect(hostedFieldOn).toHaveBeenCalledWith('inputSubmitRequest', expect.any(Function));
-            expect(hostedFieldOn).toHaveBeenCalledWith('cardTypeChange', expect.any(Function));
-            expect(hostedFieldOn).toHaveBeenCalledWith('validityChange', expect.any(Function));
-        });
-
-        it('throws an error if an element with provided card name container id is not defined', async () => {
-            document.body.removeChild(paypalCardNameFieldElement);
-
-            try {
-                await strategy.initialize(initializationOptions);
-            } catch (error) {
-                expect(error).toBeInstanceOf(InvalidArgumentError);
-            }
-        });
-
-        it('renders card name field for credit card form', async () => {
```

Comment:
We have two variations of getting `form.fields`, you can see where they are initialized here: https://github.com/bc-nick/checkout-js/blob/master/src/app/payment/hostedCreditCard/withHostedCreditCardFieldset.tsx#L76

So i think these tests are about checking this flows

Reply:
@bc-nick  I asked about that functionality. PPCP vaultings is still not implemented on the backend. So for now we do not need tests here because most likely I will have to finish that functionality. Once we will have a completed vaultings, so then we can cover it.

Reply:
@bc-dronov thanks for your investigation. Good job!